src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -53,16 +53,19 @@
  */
 public class UserController {
 
-    private static String AUTH0_DOMAIN = DataManager.getConfigPropertyAsText("AUTH0_DOMAIN");
-    private static String AUTH0_CLIENT_ID = DataManager.getConfigPropertyAsText("AUTH0_CLIENT_ID");
-    private static String AUTH0_API_TOKEN = DataManager.getConfigPropertyAsText("AUTH0_TOKEN");
+    private static final String AUTH0_DOMAIN = DataManager.getConfigPropertyAsText("AUTH0_DOMAIN");
+    private static final String AUTH0_CLIENT_ID = DataManager.getConfigPropertyAsText("AUTH0_CLIENT_ID");
+    private static final String AUTH0_API_TOKEN = DataManager.getConfigPropertyAsText("AUTH0_TOKEN");
+    static final int TEST_AUTH0_PORT = 8089;
```

Comment:
Although the PR this PR is adding to is testing just the UserController, there are also other places where Auth0 could be stubbed out - for example in testing subscription notifications. I think it may make sense to move all logic for creating Auth0 request URLs into one place - possibly in Auth0Connection.java. I don't think this belongs here.